src/Sentry.Unity.Editor/ConfigurationWindow/OptionsConfigurationTab.cs

```diff
@@ -68,23 +93,30 @@ internal static void CreateScript()
                 scriptPath = "Assets" + scriptPath.Substring(Application.dataPath.Length);
             }
 
-            var scriptName = Path.GetFileNameWithoutExtension(scriptPath);
-
-            File.WriteAllText(scriptPath, $@"using Sentry.Unity;
-using UnityEngine;
-
-[CreateAssetMenu(fileName = ""Assets/Resources/Sentry/{scriptName}.cs"", menuName = ""Sentry/{scriptName}"", order = 999)]
-public class {scriptName} : ScriptableOptionsConfiguration
-{{
-    // This method gets called when you instantiated the scriptable object and added it to the configuration window
-    public override void Configure(SentryUnityOptions options)
-    {{
-        // NOTE: Native support is already initialized by the time this method runs, so Unity bugs are captured.
-        // That means changes done to the 'options' here will only affect events from C# scripts.
+            scriptName = Path.GetFileNameWithoutExtension(scriptPath);
 
-        // Your code here
-    }}
-}}");
+            var template = new StringBuilder();
+            template.AppendLine("using System;");
+            template.AppendLine("using UnityEngine;");
+            template.AppendLine("using Sentry.Unity;");
+            if (isEditorScript)
+            {
+                template.AppendLine("using Sentry.Unity.Editor;");
+            }
+            template.AppendLine();
+            template.AppendFormat("[CreateAssetMenu(fileName = \"Assets/Resources/Sentry/{0}.cs\", menuName = \"Sentry/{0}\", order = 999)]\n", scriptName);
```

Comment:
this is what the original code did. should I try what happens if I change it to '.asset'?

Reply:
https://github.com/getsentry/sentry-unity/pull/1046/files/c14169788d1d0188f3e0d6da46d94a895dbdaf33#diff-d31bf7ab74fa7bc86eb5dd584e2fd8249cc143f376995a2429759a2298226b4dL76

Reply:
I think that was an error in the original implementation.
If I try to go to `Assets/Create/Sentry/*OneOfTheGeneratedOnes*` it errors and logs
```
CreateAsset() should not be used to create a file of type 'cs' - consider using AssetDatabase.ImportAsset() to import an existing 'cs' file instead or change the file type to '*.asset'. This error will in a future release be changed to an exception.
```
instead of instantiating a scriptable object.

Reply:
changed